os/booting-on-azure.md

```diff
@@ -59,55 +59,34 @@ etcd:
-
 ## Launching instances
 
-### Via the cross-platform CLI
-
-Follow the [installation and configuration guides][xplat-cli] for the Microsoft Azure Cross-Platform CLI to set up your local installation. This tool can be used to perform most of the needed tasks.
-
-Instances on Microsoft Azure must be connected to a cloud service. Create a new cloud service with the following command:
-
-```sh
-azure service create my-cloud-service
-```
-
-All of the instances within a cloud service can connect to one another via their private network interface. Instances within a cloud service are effectively NAT'd behind the cloud service's address; the public address. To allow connections from outside the cloud service, you'll need to create an endpoint with `azure vm endpoint create`. For now, we'll keep it simple and only connect to other machines within our cloud-service.
+### Via the Microsoft Azure CLI
 
-In order to SSH into your machine, you'll need an x509 certificate. You probably already have an SSH key, which you can use to generate an x509 certificate. More detail can be found in [this guide to ssh keys and x509 on Microsoft Azure][ssh].
+Follow the [installation and configuration guides][azure-cli] for the Microsoft Azure CLI to set up your local installation.
 
-Now that you have a cloud service and your keys, create an instance of Container Linux Alpha {{site.alpha-channel}}, connected to your cloud service:
+Instances on Microsoft Azure must be created within a resource group. Create a new resource group with the following command:
 
 ```sh
-azure vm create --custom-data=config.ign --ssh=22 --ssh-cert=path/to/cert --no-ssh-password --vm-name=node-1 --connect=my-cloud-service $(azure vm image list --json | jq --raw-output '.[].name | select(contains("__CoreOS-Stable-{{site.alpha-channel}}"))') core
+az group create --name group-1 --location <location>
 ```
 
-This will additionally create an endpoint allowing SSH traffic to reach the newly created instance. If you bring up more machines, you'll need to choose a different port for each instance.
-
-Let's create two more instances:
-
-```sh
-azure vm create --custom-data=config.ign --ssh=2022 --ssh-cert=path/to/cert --no-ssh-password --vm-name=node-2 --connect=my-cloud-service $(azure vm image list --json | jq --raw-output '.[].name | select(contains("__CoreOS-Stable-{{site.alpha-channel}}"))') core
-```
+Now that you have a resource group, create an instance of Container Linux Alpha {{site.alpha-channel}} inside it:
 
 ```sh
-azure vm create --custom-data=config.ign --ssh=3022 --ssh-cert=path/to/cert --no-ssh-password --vm-name=node-3 --connect=my-cloud-service $(azure vm image list --json | jq --raw-output '.[].name | select(contains("__CoreOS-Stable-{{site.alpha-channel}}"))') core
+az vm create --name node-1 --resource-group group-1 --admin-username core --custom-data "$(cat config.ign)" --image CoreOS:CoreOS:Alpha:{{site.alpha-channel}}
```

Comment:
Ah, interesting. I was running it within a container, so I didn't get the shorthand. I agree, we should omit it.